#!/usr/bin/env python
PACKAGE = "fuzzymar_multi_robot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Alpha_utility", double_t, 0, "Utility factor in global stimulous", 2.0 , 0.0, 20.0)
gen.add("UDD_factor", double_t, 0, "Factor that changes the curve of dynamic utility", 0.07 , -5.0, 5.0)
gen.add("Alpha_ports", double_t, 0, "Factor that changes the influence of ports in global stimulous", 1.5 , 0.0, 20.0)
gen.add("Max_vel", double_t, 0, "The max speed of the robot", 0.2 , 0.0, 2.0)
gen.add("apply_changes", bool_t, 0, "If set to True, the changes will take effect", False)

exit(gen.generate(PACKAGE, "fuzzymar_multi_robot", "kobukis"))